molecule/default/molecule.yml

```diff
@@ -20,6 +20,7 @@ platforms:
     command: /usr/sbin/init
     privileged: true
     pre_build_image: true
+    api_version: auto
```

Comment:
It is strange to add it here without justification and then remove it in the next commit without explanation. This also makes the review more complicated since it requires to review code that is not really used.